src/Framework/AspNetCoreAnalyzers/src/Analyzers/DiagnosticDescriptors.cs

```diff
@@ -196,4 +196,13 @@ internal static class DiagnosticDescriptors
         DiagnosticSeverity.Warning,
         isEnabledByDefault: true,
         helpLinkUri: "https://aka.ms/aspnet/analyzers");
+
+    internal static readonly DiagnosticDescriptor AtMostOneFromBodyAttribute = new(
+        "ASP0024",
+        "At most one parameter can have a [FromBody] attribute.",
```

Comment:
```suggestion
        "Route handler has multiple parameters with the [FromBody] attribute.",
```

Reply:
Actually the approved message was as specified above (@halter73 specifically called it out in the approval). I'm fine either way but just want to give @halter73  a chance to comment just in case he had a particular affinity to the wording as approved in the linked issue.